.helm/assure-hmrc-data/values/production.yaml

```diff
@@ -22,6 +22,7 @@ ingress:
   annotations:
     external-dns.alpha.kubernetes.io/set-identifier: "assure-hmrc-data-laa-assure-hmrc-data-production-green"
     external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/whitelist-source-range: <%= ENV['ALLOW_LIST'] %>
```

Comment:
i don't think the ENV var is available at this point?

Reply:
👍